src/ui/events.js

```diff
@@ -813,7 +814,7 @@ export type MapEvent =
      * @event movestart
      * @memberof Map
      * @instance
-     * @property {{originalEvent: DragEvent}} data
+     * @type {{originalEvent: DragEvent}}
```

Comment:
@SnailBones I'm working on the UI for this update and noticed that this event type is formatted differently than others. Should this type be: 

```suggestion
     * @type {DragEvent}
```

Reply:
Good catch! I was running Regex over them to convert them but it looks like I missed that one!